.github/scripts/matrices.py

```diff
@@ -99,12 +99,13 @@ def __init__(
         n_partitions=2,
         pr_cross_platform=False,
     ),
-        Case(
-        name="integration / polkadot_localnode",
-        filter="(package(=cast) | package(=forge)) & test(/polkadot_localnode/)",
-        n_partitions=1,
-        pr_cross_platform=False,
-    ),
+    # TODO: run the local node tests on polkadot-anvil
```

Comment:
in this file, you need to replace

`t_macos = Target("parity-macos", "aarch64-apple-darwin", "macosx-aarch64")`

with

`t_macos = Target("macos-latest", "aarch64-apple-darwin", "macosx-aarch64")`

This is the reason why all CI tests on mac are failing right now